website/docs/os.is.md

```diff
@@ -4,6 +4,8 @@ Checks the current operating system identifier against a particular value.
 os.is("id")
 ```
 
+**This function has been deprecated.** Use [os.target()](os.target.md) or [os.host()](os.host.md) instead.
```

Comment:
Maybe we could use fancy-dancy [Admonitions](https://premake.github.io/docs/Style-Guide/#admonitions) for the first time on our website? 😀

It would be:
```
:::caution
**This function has been deprecated.** Use [os.target()](os.target.md) or [os.host()](os.host.md) instead.
:::
```

I've tested it on my local demo branch for `configuration`: https://kyriets.github.io/premake-core/docs/configuration/

What do you think?

Reply:
Ah yes, I was actually looking for something like this.

Happy to make this change and also make a separate PR that uses them for all deprecation notices?

Reply:
On second thoughts, maybe it's clearer to leave this PR as is and make a new one that does the change everywhere

Reply:
I think we use it here now, in case no one gets around to the follow up PR to do it everywhere immediately.